build.sbt

```diff
@@ -166,3 +169,12 @@ lazy val `stripe-intent` = (project in file("support-lambdas/stripe-intent"))
 
 lazy val `support-redemptiondb` = (project in file("support-redemptiondb"))
   .enablePlugins(RiffRaffArtifact)
+
+lazy val `it-test-runner` = (project in file("support-lambdas/it-test-runner"))
+  .enablePlugins(JavaAppPackaging, RiffRaffArtifact)
+//  .configs(IntegrationTest)
+  .settings(
+//    commonSettings,
+//    integrationTestSettings,
+//    libraryDependencies ++= commonDependencies
+  ).dependsOn(`support-services`)// todo very excessive
```

Comment:
we only need the cloudwatch metric put code so this dependency really bloats the lambda jar.  this will be probably be a copy and paste job for simplicity.